src/Microsoft.AspNetCore.Mvc.Formatters.Xml/XmlDataContractSerializerInputFormatter.cs

```diff
@@ -48,6 +48,16 @@ public XmlDataContractSerializerInputFormatter()
 
             WrapperProviderFactories = new List<IWrapperProviderFactory>();
             WrapperProviderFactories.Add(new SerializableErrorWrapperProviderFactory());
+
```

Comment:
Suggest this list of factories is getting long enough to put it in one place instead of four.